app/client/templates/exhibitor/payment.js

```diff
@@ -0,0 +1,78 @@
+Template.payment.created = function () {
+  //
+};
+
+Template.payment.helpers({
+  //
+});
+
+Template.payment.rendered = function () {
+  $('select').material_select();
+
+  Session.set('paymentErrors', 0);
```

Comment:
Needs this tracker auto run and markup for the alert: https://github.com/interglobalvision/app-material-fair-com/blob/master/app/client/templates/application/signed.js#L8